apps/spark/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'openhouse.java-conventions'
     id 'openhouse.hadoop-conventions'
-    id 'openhouse.iceberg-conventions'
+    id 'openhouse.iceberg-conventions-1.2'
```

Comment:
Why this module is using iceberg 1.2?

Reply:
We are only upgrading the server for now.